spring-cloud-azure-config/src/main/java/com/microsoft/azure/spring/cloud/config/domain/KeyValueItem.java

```diff
@@ -10,11 +10,12 @@
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
+import lombok.ToString;
 
 import java.util.Date;
 
 @NoArgsConstructor
-@Getter @Setter
+@Getter @Setter @ToString
```

Comment:
It seems that convention is each annotation one line

Reply:
Updated with one annotation one line.